### ISSUE TITLE
Verschiedene Fehler vom Zend-Framework

### ISSUE THREAD

**Issue by Erde244** [tkrebs/ep3-bs#531]
Hallo,

wir wollen das ep3-Buchungssystem innerhalb unsere Firma zur Raum-Verwaltung nutzen und haben bereits eine entsprechende Instanz installiert. Bei mehreren Tests von Buchungen kommen bei uns jedoch immer wieder Zend-Framework-Fehler. Nach folgend die Details:

Buchung erfolgt über einen Account mit der Berechtigung "aktivierter Benutzer". Nach der Auswahl des Timeslots und der Bestätigung, dass man diesen buchen möchte erscheinen die Fehlermeldungen lt. Anhang.

![zend05](https://user-images.githubusercontent.com/98869860/152105499-39403029-592c-4a50-a6b6-e8cfb9b7c46a.png)
![zend04](https://user-images.githubusercontent.com/98869860/152105500-014f675c-58e1-4869-8793-41ae252340df.png)
![zend03](https://user-images.githubusercontent.com/98869860/152105502-37289bf9-2af4-48ae-b762-517ce8885bff.png)
![zend02](https://user-images.githubusercontent.com/98869860/152105503-082e1bf3-b0b1-4592-a860-f6e17bf2520a.png)
![zend01](https://user-images.githubusercontent.com/98869860/152105506-208539eb-731f-4fae-b838-253274804614.png)

Wenn man danach zurück auf die Startseite geht, wurde keine Buchung vermerkt.

Kann uns hier jemand weiterhelfen?

MfG
Marcus

**Comment by cbits68** [comment]
sieht so aus, als wenn der Mailservice bei der Buchung eines Users nicht funktioniert.
Wie sieht es denn aus, wenn Ihr eine Buchung im Backend als Admin macht? Läuft die durch?

**Comment by Erde244** [comment]
Bei der Buchung über das Backend funktioniert es ohne Fehlermeldung. Im ersten Schritt wollen wir den Mail-Service nicht nutzen. Kann man das deaktivieren?

**Comment by cbits68** [comment]
es ist sicher möglich den Mailservice im Buchungsprozess herauszunehmen (auskommentieren).
Dann aber auch bei Storierungen nicht vergessen. 
Ich denke es ist aber aufwändiger als mal zu schauen, wo Eure Konfiguration des Mailservice  ggf. fehlerhaft ist oder Euer System damit nicht funktioniert. Ist ja nur die local.php

**Comment by Erde244** [comment]
Hallo,

wir haben es jetzt erneut mit mehreren Mail-Einstellungen getestet. Der Hauptanteil der Fehler ist verschwunden. Ein Fehler wird aber immer noch angezeigt. Dieser könnte aber ggf. auch von fehlenden Firewall-Einstellungen kommen. 

![zend06](https://user-images.githubusercontent.com/98869860/152287135-0c1e0786-a0ed-468f-af8b-e683f78f289f.png)

Könnte dies daher kommen? Welche konkreten Variablen müssen den für die Version "sendmail" bzw. "smt-tls" hinterlegt werden. Vielleicht haben wir dort auch noch etwas verwechselt.

MfG Marcus

**Comment by cbits68** [comment]
In der Fehlermeldung wird auf die Connection.php referenziert. Das hat vermutlich eher etwas mit der Verbindung zur Datenbank zu tun und nicht mit den Mail Einstellungen.
Ich hoffe Ihr habt bei der Installation auch darauf geachtet, dass Ihr die richtige PHP Version nutzt.

**Comment by Erde244** [comment]
Wir haben die PHP-Version 7.4.27 im Einsatz, was dann eigentlich auch passen sollte.

**Comment by cbits68** [comment]
ja, das sollte passen. Aber irgendwo ist wohl noch ein Problem wenn mittels POD (PHP Data Objects) beim Buchen auf die MySQL DB zugegriffen wird.

**Comment by Erde244** [comment]
Merkwürdigerweise wird der Eintrag in der Datenbank trotzdem vorgenommen. Wenn wir bei einem Nutzer-Account die Mail-Benachrichtigung deaktivieren, kommt keine Fehlermeldung, daher weiterhin der Verdacht bzgl. Mailversand.

**Comment by cbits68** [comment]
Und werden Mails versendet oder nicht?
